src/app/core/entity-components/entity-subrecord/entity-subrecord/entity-subrecord.component.ts

```diff
@@ -224,9 +228,7 @@ export class EntitySubrecordComponent<T extends Entity>
   ngOnChanges(changes: SimpleChanges) {
     if (changes.hasOwnProperty("columns")) {
       this.initFormGroups();
-      if (this.columnsToDisplay.length < 2) {
```

Comment:
as unintuitive as this check looks, it is necessary - otherwise displayColumns provided by config are lost ...